message_definitions/v1.0/messages.xml

```diff
@@ -696,6 +696,7 @@
   <message name="MISSION_STATUS" id="79">
     <field name="remaining_time" type="float"/>
     <field name="task_list" type="uint8[]"/>
+    <field name="index_list" type="uint8[]"/>
```

Comment:
Unfortunately, it is not possible to have two variable-length arrays in a message. Only one placed at the end is allowed. In this case, the index_list probably gives more information than the task_list.